packages/block-library/src/navigation-link/editor.scss

```diff
@@ -64,6 +64,12 @@
 		margin-bottom: $grid-unit-20;
 		margin-left: $grid-unit-20;
 	}
+
+	// Override the list reset for colored, nested menus
```

Comment:
Could we add to the comment "why" this override is required?

Reply:
It was supposed to fix a color blowout, but that was fixed elsewhere and I was able to remove the code in f04646fb3f